### DISCUSSION TITLE
Ternoa Alphanet Testing

### DISCUSSION THREAD

**Discussion by markopoloparadox** [capsule-corp-ternoa/ternoa-hub#28] [Announcements]
Hello all.

We are currently doing some extreme testing on Alphanet where we test how our chain works in cases of chain halt or chain attack. There is nothing to worry about, the goal is to see if we can make it work using existing extrinsics or if we will need to activate the emergency protocol.

This exercise should give us a better understanding and insight on how to react when an attack would happen on Mainnet.

**Comment by Crypte** [comment]
Hello Marko, 
I can't wait to see the results of these tests. I hope they are encouraging!

**Comment by Orius** [comment]
Hi Marko.
When deploying a network that must run without interruption, it is normal to implement a disaster recovery plan (DRP) in case of problems that inhibit the architecture from producing.
The case of a Datacenter failure or incident is part of the list of impediments. A DPR must be able to give a response to each case, including severe incidents (attack or hack, flood, earthquake, nuclear, etc ...).
At Orius, we have established a DPR for our servers for several years.
It's really a good thing that Ternoa is doing this for its blockchain, with a resilience solution that would allow to switch to the Alphanet.
Looking forward to the results of this test, and more generally of the Ternoa DPR 💪
